functions/src/modules/SE.functions.ts

```diff
@@ -186,3 +186,92 @@ export const createTaskFromChunks = functions.database.ref(
   });
   return snapshot.val();
 });
+
+
+
+/**
+ * Sends a notification email to the coordinator & udpates the corresponding Task
+ */
+const seUploadsBucketUrl = functions.config().sound_editing.uploads.bucket_name;
+export const processUploadedFile = functions.storage.bucket(seUploadsBucketUrl).object()
```

Comment:
Construct the uploads bucket name here according to https://trello.com/c/59mJXDBs/73-storage-structure.